scripts/viz/param_sweep_viz.R

```diff
@@ -0,0 +1,74 @@
+# Pan-Cancer Variational Autoencoder
+# Gregory Way 2017
+#
+# Visualizing results of parameter sweep
+
+library(readr)
+library(ggplot2)
+library(dplyr)
+
+param_sweep_file <- file.path("results", "parameter_sweep_full_results.tsv")
+param_df <- readr::read_tsv(param_sweep_file,
+                            col_types = cols(train_epoch = col_integer(),
+                                             loss = col_double(),
+                                             val_loss = col_double(),
+                                             learning_rate = col_character(),
+                                             batch_size = col_character(),
+                                             epochs = col_character(),
+                                             kappa = col_character()))
+
+param_melt_df <- reshape2::melt(param_df,
+                                id.vars = c("learning_rate", "batch_size",
+                                            "epochs", "kappa", "train_epoch"),
+                                measure.vars = c("loss", "val_loss"),
+                                variable.name = "loss_type",
+                                value.name = "loss")
+
+param_melt_df$batch_size <- factor(param_melt_df$batch_size,
+                                   levels = c(50, 100, 128, 200))
+param_melt_df$epochs <- factor(param_melt_df$epochs,
+                               levels = c(10, 25, 50, 100))
+
+# Sweep over measurements of kappa with batch_size and epochs as facets
+for (k in unique(param_melt_df$kappa)) {
+  subset_param <- param_melt_df %>% dplyr::filter(kappa == k)
+  
+  p <- ggplot(subset_param, aes(x = train_epoch, y = loss)) +
```

Comment:
So I think these are the plots included in [`parameter_sweep.md`](https://github.com/greenelab/tybalt/blob/master/parameter_sweep.md) - is that correct? The fact that the "row" panels are batch sizes is not labeled in those plots and is not immediately obvious in the absence of a figure legend. I might recode these so it's clear what these facets are showing. For instance:

```R
param_melt_df$batch_size <- 
  factor(param_melt_df$batch_size, levels = c(50, 100, 128, 200))
param_melt_df$batch_size <- 
  dplyr::recode_factor(param_melt_df$batch_size, 
                       `50` = "batch: 50", 
                       `100` = "batch: 100",
                       `128` = "batch: 128",
                       `200` = "batch: 200") 
```

Might even do something similar for the epochs, but at least the x-axis label `train_epoch` is a pointer for what's going on.